src/objectsAdvanced/task4.js

```diff
@@ -0,0 +1,6 @@
+const fn = (obj, field, value) => {
+    const newObj = { ...obj, surname: 'lecturer' }
```

Comment:
You are going to the right direction, just you don't use function arguments: field, and value at all. They were passed to the function, with the purpose to identify which field should be overridden and with what value.
Try to improve!